cli/src/lib/analyze/block.ts

```diff
@@ -1,27 +1,27 @@
 import assert from "assert";
-import { Match } from "./match";
-import { Selection } from "../util/selection";
+import { PairedOccurrence } from "./pairedOccurrence";
+import { Region } from "../util/region";
 import { Range } from "../util/range";
 
 /**
- * A fragment is a collection of one or more consequent matches (kmers).
+ * A block is a collection of one or more consequent pairedOccurrences (kmers).
  *
- * A fragment can be extended with a new match if its kmer indices in both
- * files are directly after that of the fragment.
+ * A block can be extended with a new match if its kmer indices in both
+ * files are directly after that of the block.
  */
-export class Fragment {
+export class Block {
 
-  public matches: Array<Match>;
+  public pairedOccurrences: Array<PairedOccurrence>;
```

Comment:
I would use shorter variable and function names for `PairedOccurence`s. Either `occurrence` or `pair` maybe? What makes most sense for you?

Reply:
I think that pair will make the most sense in a lot of cases.